.github/workflows/build-test.yml

```diff
@@ -0,0 +1,32 @@
+name: build-test
```

Comment:
I separated these into separate workflows so I can show badges for build-test, versions and proxy status on the readme.  In general I like the separation.